src/hearings/components/hearing-judge-name/hearing-judge-name.component.scss

```diff
@@ -0,0 +1,10 @@
+:host ::ng-deep .govuk-form-group {
+  .govuk-hint{
+    font-weight: bold;
+    color: #0b0c0c;
+    margin-bottom: 5px;
+  }
+  .govuk-input{
+    width: 80% !important;
```

Comment:
You should not use important try using a class to overide it

Reply:
Code is written in the library and their !important is used. So, used to override styles.